### ISSUE TITLE
Create a github pages from this repo

### ISSUE THREAD

**Issue by Jo-Schie** [openkfw/open-geodata-model#2]
@fretchen : do you have expirience with github pages. It would be nicer since we could provide content in a user-frindly way and have the link from the RMMV Guidebook to the page... If this is a quick thing to do we might consider it.

**Comment by fretchen** [comment]
I really like [mkdocs-material](https://squidfunk.github.io/mkdocs-material/). This is also what we used for the [d4dtools](https://github.com/openkfw/d4dtools) and I must admit that I do not see any problems with it until now...

The easiest way would be to put all the important information into an `index.md` file and then we can use this as the basis for the website...

**Comment by Jo-Schie** [comment]
Yeah... I guess that would be the logical conclusion of all of this effort, although by using markdown we might not make it totally easy for non-technical people to update the documentation. 

But anyways using GitHub we already embark on that journey meaning that updates to the docs and documentation will have to be made, or at least assisted, by someone who knows how to use GitHub and markdown. 

Would be great if you could have a look how to transform it. I think there are tools to do this e.g. pandoc see [here](https://medium.com/geekculture/how-to-easily-convert-word-to-markdown-with-pandoc-4d60878ccc64)...
